tests/specs/api.e2e.js

```diff
@@ -15,12 +15,14 @@ describe('newrelic api', () => {
     })
     const agentInstanceApiMethods = await browser.execute(function () {
       function getAllPropertyNames (obj) {
-        let result = new Set()
+        var result = new Set()
         while (obj) {
-          Object.getOwnPropertyNames(obj).forEach(p => result.add(p))
+          Object.getOwnPropertyNames(obj).forEach(function (p) {
+            return result.add(p)
+          })
           obj = Object.getPrototypeOf(obj)
         }
-        return [...result]
+        return Array.from(result)
```

Comment:
ICR Do we have a polyfill for this already for ie11?

Reply:
Yes.